.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -20,20 +16,20 @@ jobs:
         CONFIG: linux_64_numpy1.22python3.9.____73_pypy
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
-      linux_64_numpy1.22python3.9.____cpython:
-        CONFIG: linux_64_numpy1.22python3.9.____cpython
+      linux_64_numpy1.26python3.12.____cpython:
```

Comment:
Why did thin only go to 1.26 rather than 2.0?

Reply:
See discussion [here](https://github.com/conda-forge/conda-forge.github.io/issues/1997#issuecomment-2094361188). We're a bit boxed in with several constraints here. While we decide how to deal with this, I'm turning this PR into a draft.

Reply:
It's worth noting this PR ( https://github.com/conda-forge/matplotlib-feedstock/pull/377 ) was already merged, but that is in the `testing` label. We will want to refresh that whenever the migrator issues are sorted out

Reply:
I was enthusiastic and more YOLOy about the testing label.

Reply:
Yeah totally reasonable

More noting that we have todo there. Thankfully today that should be relatively easy to do with a re-render

Reply:
I don't see why the testing branch has to be kept in sync at all, but then, I'm not a maintainer here. In any case, it's unrelated to this PR, which I'd like to merge, so reviews welcome.